lib/rules/shebang.js

```diff
@@ -13,7 +13,7 @@ const getNpmignore = require("../util/get-npmignore")
 
 const NODE_SHEBANG = "#!/usr/bin/env node\n"
 const SHEBANG_PATTERN = /^(#!.+?)?(\r)?\n/u
-const NODE_SHEBANG_PATTERN = /#!\/usr\/bin\/env node(?: [^\r\n]+?)?\n/u
+const NODE_SHEBANG_PATTERN = /#!\/usr\/bin\/env(?: -S)? node(?: [^\r\n]+?)?\n/u
```

Comment:
mmm, there are other flags for `env` it may be worth adding support for them too.
One I used recently was `env -i` to execute something in an empty env :thinking:

Maybe adding a little looser check would be better? eg:
```suggestion
const NODE_SHEBANG_PATTERN = /#!\/usr\/bin\/env(?: .*?)? node(?: [^\r\n]+?)?\n/u
```

---

What do you think?

Reply:
Yeah, that would make sense. I'll update and add some test cases.